app/notifications/validators.py

```diff
@@ -99,17 +103,72 @@ def check_service_over_daily_message_limit(key_type: ApiKeyType, service: Servic
     counter_name="rate_limit.live_service_daily_sms",
     exception=LiveServiceTooManySMSRequestsError,
 )
-def check_service_over_daily_sms_limit_and_warn(key_type: ApiKeyType, service: Service):
-    if current_app.config["FF_SPIKE_SMS_DAILY_LIMIT"] and key_type != KEY_TYPE_TEST and current_app.config["REDIS_ENABLED"]:
-        fragments_sent = fetch_daily_sms_fragment_count(service.id)
-        warn_about_daily_sms_limit(service, fragments_sent)
+def check_sms_daily_limit(service: Service, requested_sms=0):
+    messages_sent = fetch_todays_requested_sms_count(service.id)
+    over_sms_daily_limit = (messages_sent + requested_sms) >= service.sms_daily_limit
```

Comment:
This defines being over the limit when you are actually at the limit.  It prevents a user from sending their last notification and stops them from getting the email that let's them know they've reached their limit

```suggestion
    over_sms_daily_limit = (messages_sent + requested_sms) > service.sms_daily_limit
```

Reply:
Hmm good point. I think what we actually want is to block them from sending more than the limit but warn them they are at the limit when they are at the limit. I'll adjust.